customize_image.sh

```diff
@@ -64,12 +67,15 @@ OUTPUT_IMAGE=${2:-$1}
 
 if [ "$SOURCE_IMAGE" == "$OUTPUT_IMAGE" ]; then
 	echo "WARNING: No name for the output image was specified. The source image will be modfied in-place."
-	echo ""
-	echo -n "Do you want to continue? [Ny] "
-	read -r choice
 
-	if ! [[ "$choice" == "y" || "$choice" == "y" ]]; then
-		exit
+	if [ $FORCE == 0 ]; then
```

Comment:
Shouldn't this be -eq as it is an arithmetic comparison?

Reply:
You're right. Fixes this. Thanks.